src/System.Text.Json/src/System/Text/Json/JsonTokenType.cs

```diff
@@ -15,17 +15,68 @@ public enum JsonTokenType : byte
     {
         // Do not re-order.
         // We rely on the ordering to quickly check things like IsTokenTypePrimitive
+
+        /// <summary>
+        ///   Indicates that there is no value (as distinct from <see cref="Null"/>).
+        /// </summary>
+        /// <remarks>
+        ///   This is the default token type if no data has been read by the <see cref="Utf8JsonReader"/>.
+        /// </remarks>
         None,
+
+        /// <summary>
+        ///   Indicates that the token type is the start of a JSON object.
+        /// </summary>
         StartObject,
+
+        /// <summary>
+        ///   Indicates that the token type is the end of a JSON object.
+        /// </summary>
         EndObject,
+
+        /// <summary>
+        ///   Indicates that the token type is the start of a JSON array.
+        /// </summary>
         StartArray,
+
+        /// <summary>
+        ///   Indicates that the token type is the end of a JSON array.
+        /// </summary>
         EndArray,
+
+        /// <summary>
+        ///   Indicates that the token type is a JSON property name.
+        /// </summary>
         PropertyName,
+
+        /// <summary>
+        ///   Indicates that the token type is a JSON string.
+        /// </summary>
         String,
+
+        /// <summary>
+        ///   Indicates that the token type is a JSON number.
+        /// </summary>
         Number,
+
+        /// <summary>
+        ///   Indicates that the token type is the JSON literal <c>true</c>.
+        /// </summary>
         True,
+
+        /// <summary>
+        ///   Indicates that the token type is the JSON literal <c>false</c>.
```

Comment:
Reserved keywords like `null`, `true` and `false` should be enclosed by `<see langword="keyword" />`:

```suggestion
        ///   Indicates that the token type is the JSON literal <see langword="false" />.
```

Reply:
See the comment regarding true/false/null above. https://github.com/dotnet/corefx/pull/38037#discussion_r288844749